### PR TITLE
updated copy; closes #4

### LINKED ISSUE #4: [www] three feature sections
### V1 With a focus on discoverability

**Screenshot**
`please login aws primary`

**Blurb**
It can be a nightmare managing access to all of the services your team uses, much less making sure they know how to get to them all. PleaseOps makes it safe and simple for your team to access your infrastructure. It generates federated login URLs for supported services, such as AWS, and provides a lookup directory for custom services, so you can get rid of all those outdated bookmarks.

### V2 With a focus on access control

**Screenshot**
`please grant user jsmith permission "login aws primary"`

**Blurb**
It can be a nightmare managing who has access to what across several services, not to mention onboarding and offboarding. With PleaseOps, you can avoid the problem of rotating credentials by never provisioning employee-specific credentials in the first place. Granting access to a new employee is as simple as `please grant user jsmith role developer`, and removing an employee is as simple as `please remove user jsmith`.

## Section 2: Manage Operations
**Screenshot**
`please deploy jenkins myapp`

**Blurb**
Instead of adding notification integrations to ensure everyone knows when the last deploy happened, kick off code deployments with a chat message that the rest of the team can see. PleaseOps makes it easy to manage your operations from a single, visible interface.

## Section 3: Triage Issues
**Screenshot**
`please check kubernetes prod-cluster`

**Blurb**
When something goes wrong, it's important to make sure everyone is on the same page. With PleaseOps, your team can collaboratively triage issues within the same chat conversation. No more chasing down the right screenshot or link to send; instead, you can pull relevant information into the conversation with a simple command.

### PR DESCRIPTION
**Pull request by thuey**

### CODE REVIEW

**Comment by thuey** [comment]
I'll add the screenshots before merging this in

File: layouts/index.html [hunk 1]
```diff
@@ -4,8 +4,8 @@
         <div class="row">
             <div class="col s12 m6">
                 <div class="section">
-                    <h5>Simplify Your Operations</h5>
-                    Access all of your operations tools from a single cli or chat. <code>please login to aws read-only</code> or <code>please get kubernetes app</code> is all you need to get started.
+                    <h5>Access Infrastructure</h5>
+                    PleaseOps generates federated login URLs for supported services, such as AWS, and provides a lookup directory for custom services. Granting access is as simple as <code>please grant role developer permission "login aws primary"</code> and then <code>please grant user jsmith role developer</code>.
```
Comment (jonnymbgood) [inline layouts/index.html]: You'll need to escape the `"`s

File: layouts/index.html [hunk 2]
```diff
@@ -32,8 +32,8 @@ <h5>Collaborate</h5>
         <div class="row">
             <div class="col s12 m6">
                 <div class="section">
-                    <h5>Onboard in Seconds</h5>
-                    No more installing your ops tooling on everyone's machines and trying to grant just the right amount of access for that new hire. Just <code>please set user jdoe roles to developer</code> and they have everything they need to get started available through chat.
+                    <h5>Triage Issues</h5>
+                    When something goes wrong, it's important to make sure everyone is on the same page. Commands like <code>please check k8s cluster</code>, <code>please check k8s cluster ns</code>, and <code>please check service aws</code> pull relevant information into a conversation the entire team can collaborate on.
```
Comment (jonnymbgood) [inline layouts/index.html]: I think you need to escape the `'`s too.

Comment (jonnymbgood) [inline layouts/index.html]: Maybe `please check k8s cluster myapp`?

File: layouts/index.html (hunk 1 above)
Comment (thuey) [reply layouts/index.html]: Hmmm... Why do you think I need to escape these?

image

File: layouts/index.html (hunk 2 above)
Comment (thuey) [reply layouts/index.html]: ?

image

**Comment by thuey** [comment]
Hmmm... Not a huge fan of how this first rough attempt at screenshots went:

File: layouts/index.html (hunk 1 above)
Comment (jonnymbgood) [reply layouts/index.html]: NM. That's only in attributes. I've gotten so used to escaping things I forgot that `<>&` are the only reserved characters in html outside of attributes.

**Comment by jonnymbgood** [comment]
I think it would look better if the screenshots were ~1.5-2x taller than the text blocks, and the text blocks center aligned with their images.

We could alternate colors on the background, and have a dark theme screenshot on light, and a light themed screenshot on dark. (hero dark bg, section 1 light bg w/ dark theme screenshot, section 2 darkish/grey background w/ light theme screenshot, etc... If memory serves slack has a dark theme, and you can use iterm or a dark themed console for the cli shots.

**Comment by thuey** [comment]
If it's okay with you, I'd like to call this one good enough for now. You can check it out here: https://www.pleaseops.com/

**Pull request merged**